chain/messagepool/messagepool.go

```diff
@@ -303,6 +304,7 @@ func (ms *msgSet) add(m *types.SignedMessage, mp *MessagePool, strict, untrusted
 	ms.msgs[m.Message.Nonce] = m
 	ms.requiredFunds.Add(ms.requiredFunds, m.Message.RequiredFunds().Int)
 	// ms.requiredFunds.Add(ms.requiredFunds, m.Message.Value.Int)
+	stats.Record(ctx, metrics.MessagePending.M(int64(len(mp.pending))))
```

Comment:
This is gonna record the number of entries in `mp.pending`, which is the total number of senders that currently have messages in our mpool. 

Is that what we want to record here, or do we want the total number of messages in our mpool?

Reply:
Total number of messages! What do I need to record that instead?

Reply:
@GlacierWalrus Poking around the code, it looks like `allPending()` seems to do what we want here. 

This is NOT gonna be a fast method, though, so we either want to make this metric opt-in (if it isn't already), or only capture it infrequently.

Reply:
@arajasek I've made updates as you suggested, but I'm having trouble working out how to configure mpool. I would be grateful if you could advise here. Configuring seems to work with `lotus mpool config '{...}'` but it should be configurable from the lotus config file/env vars.